compute/compute-node.Dockerfile

```diff
@@ -1283,6 +1277,7 @@ FROM alpine/curl:${ALPINE_CURL_VERSION} AS exporters
 ARG TARGETARCH
 # Keep sql_exporter version same as in build-tools.Dockerfile and
 # test_runner/regress/test_compute_metrics.py
+RUN echo -e "--retry-connrefused\n--connect-timeout 15\n--retry 5\n--max-time 300\n" > /root/.curlrc
 RUN if [ "$TARGETARCH" = "amd64" ]; then\
```

Comment:
Can you stack it with the next command please to avoid creating extra layer?

Reply:
thanks for pointing, fixed!